thrift/src/main/thrift/contentatom.thrift

```diff
@@ -49,8 +48,8 @@ union AtomData {
   5: cta.CTAAtom cta
   6: interactive.InteractiveAtom interactive
   7: review.ReviewAtom review
-  8: recipe.RecipeAtom recipe
-  // 9: storyquestions.StoryQuestionsAtom storyquestions DEPRECATED
+//8: recipe.RecipeAtom recipe
```

Comment:
Might be wise to add the DEPRECATED indicator here too

Reply:
@rowannekabalan could I suggest also putting in when it was deprecated, who it was deprecated by (i.e. capi team) and why (because we are moving to recipes as elements)? or is that a bit too much do you think?

Reply:
Done and done @JustinPinner @fredex42